packages/cursorless-engine/src/processTargets/modifiers/EveryScopeStage.ts

```diff
@@ -143,7 +143,7 @@ function getScopesOverlappingRange(
   return Array.from(
     scopeHandler.generateScopes(editor, start, "forward", {
       distalPosition: end,
-      skipAncestorScopes: true,
+      includeDescendantScopes: true,
```

Comment:
This is necessary if we want to use ordinals with short paint

Reply:
I guess the other option is to either yield the full range or the sub ranges not both. We could make this decision based on the position. Probably something we need to discuss.

Reply:
This made a lot of other test fails I'm restoring it for now.